docs/tutorials/hello-world.md

```diff
@@ -113,22 +108,19 @@ battery. Having a thread running and trying to make network requests while the a
 to allow the user to call that function only at the right time.
 
 ```rust
-use bdk::blockchain::noop_progress;
+use bdk::SyncOptions;
 
 // ...
 
-wallet.sync(noop_progress(), None)?;
+wallet.sync(&blockchain, SyncOptions::default())?;
 ```
 
-In this case, we are not interested in receiving updates about the progress, and we just want to use the default settings, so we use [`noop_progress()`][noop_progress] and `None` as arguments. This will make queries to the Electrum server
-and store the list of transactions and UTXOs in our `Database`. In this case, we are using a `MemoryDatabase`, so those data are only going to be kept in RAM and dropped once our `Wallet` is dropped. This is very useful
+This will make queries to the Electrum server and store the list of transactions and UTXOs in our `Database`. In this case, we are using a `MemoryDatabase`, so those data are only going to be kept in RAM and dropped once our `Wallet` is dropped. This is very useful
```

Comment:
```suggestion
The `SyncOptions` determines some sync time behaviors, like progress update, etc. For this case the `default` sync option with no progress update is adequate. This will make queries to the Electrum server and store the list of transactions and UTXOs in our `Database`. In this case, we are using a `MemoryDatabase`, so those data are only going to be kept in RAM and dropped once our `Wallet` is dropped. This is very useful
```